### PR TITLE
Feature/api creation with FastAPI

### PR DESCRIPTION
**Pull request by hsauve-scottlogic**
- Created FastAPI app
- Created new task for requirements.txt

### CODE REVIEW

File: backend/api.py [hunk 1]
```diff
@@ -1 +1,11 @@
 # TODO: Add basic FastAPI app for communicating with the front end
+
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/health")
+async def health_check():
+    return {"message": "Infer GPT is running!"}
```
Comment (harryBedfordSL) [inline backend/api.py:10]: v. minor question, but will pep-8 implement whitespacing rules?

Comment (Ch0live) [reply backend/api.py:10]: We can configure it not to if it's annoying, but to start I think it will

Comment (Ch0live) [reply backend/api.py:10]: It's not set up yet however so one to re-look at this when the time comes

**Review by bheyman-scottlogic — COMMENTED** [review]
Looks good. One thing is the mention in the description of changes to requirements.txt. I cannot see any changes made to this. Was wondering if this should be included in the PR?

File: backend/api.py (hunk 1 above)
Comment (harryBedfordSL) [reply backend/api.py:10]: yeah I'm keen for it to enforce consistency when it is set up, cool thanks will approve

**Review by harryBedfordSL — APPROVED** [review]

**Comment by Ch0live** [comment]
We've made a new ticket internally for the requirements.txt work

**Review by bheyman-scottlogic — APPROVED** [review]

**Pull request merged**